examples/imgui_controls.py

```diff
@@ -0,0 +1,37 @@
+import cmap
+import numpy as np
+
+import scenex as snx
+from scenex.imgui import add_imgui_controls
+
+X, Y = np.meshgrid(np.linspace(-10, 10, 100), np.linspace(-10, 10, 100))
+view = snx.View(
+    blending="default",
+    scene=snx.Scene(
+        children=[
+            snx.Image(
+                data=np.random.randint(0, 255, (200, 200)).astype(np.uint8),
+                cmap=cmap.Colormap("viridis"),
+                transform=snx.Transform().scaled((1.3, 0.5)).translated((-40, 20)),
+                clims=(0, 255),
+                opacity=0.7,
+            ),
+            snx.Points(
+                coords=np.random.randint(0, 200, (100, 2)).astype(np.uint8),
+                size=5,
+                face_color=cmap.Color("coral"),
+                edge_color=cmap.Color("purple"),
+                transform=snx.Transform().translated((0, -50)),
+            ),
+            snx.Image(
+                name="sine image",
+                data=(np.sin(X) * np.cos(Y)).astype(np.float32),
+                clims=(-1, 1),
+            ),
+        ]
+    ),
+)
+
+canvas = snx.show(view)
+add_imgui_controls(view)
```

Comment:
Could just add this line (commented out) over in `basic_scene.py` if you don't want to duplicate the file...but maybe we don't want to include optional dependency stuff in there? Idk, doesn't really matter, would just save a file.